src/modules/template-details/TemplateDetails.tsx

```diff
@@ -56,105 +59,129 @@ const TemplateDetails = ({ match }: TemplateDetails.Props) => {
     setConfirmDeleteOpen(false);
   }, []);
 
+  const forkTemplateHandler = async () => {
+    info.incrementForks();
+    setForkInProgress(true);
+    forkTemplate(match.params.id).then(
+      (response) => console.log("Response: " + response.data),
+      (reason) => console.log("Reason: " + reason)
+    ).then(
+      () => {
+        setForkInProgress(false);
+        info.completeFork(match.params.id, false);
+        info.decrementForks();
+      }
+
+    );
+  }
+
   const patternsNames = useMemo(toNames(template), [template]);
 
   return (
     <div className={csx.templateDetails}>
       {loading ? (
         <Loader />
       ) : (
-        !error && (
-          <>
-            {confirmDeleteOpen && (
-              <ConfirmDelete template={template} onClose={closeConfirmDelete} />
-            )}
-
-            <header>
-              <NavLink to={`${match.url}/documentation`}>
-                <Button>
-                  <MenuBookIcon /> DOCS
+          !error && (
+            <>
+              {confirmDeleteOpen && (
+                <ConfirmDelete template={template} onClose={closeConfirmDelete} />
+              )}
+
+              <header>
+                <NavLink to={`${match.url}/documentation`}>
+                  <Button>
+                    <MenuBookIcon /> DOCS
                 </Button>
-              </NavLink>
+                </NavLink>
+
+                <Button disabled={forkInProgress} onClick={forkTemplateHandler}>
 
-              <a href={template.githubLink} target="_blank">
-                <Button>
-                  <ShareIcon /> SOURCE
+                  <img src="https://www.iconsdb.com/icons/preview/white/fork-2-xxl.png"
```

Comment:
This should be Icon as before imported from material UI as component - in task description you have name of this icon

Reply:
Here is a little problem, cause vs code cannot find this module. I tried reinstall @material-ui/icons, but without result.
![image](https://user-images.githubusercontent.com/34678770/97156550-24bf5c80-1777-11eb-849b-45e8dd1c95c2.png)
![image](https://user-images.githubusercontent.com/34678770/97156566-29841080-1777-11eb-9a1b-729ab7882434.png)